src/app/App.js

```diff
@@ -1,4 +1,24 @@
-export const App = ({options}) => {
-}
+export const App = ({ options }) => {
+  const buttonRanking = document.querySelector('.button--ranking');
```

Comment:
bardziej po ang jest rankingButton, można skrócić do rankingBtn :)

Reply:
ok, spoko :) myślałam, że konwencja jest raczej taka, że wpierw opisuje się funkcjonalność (button), a kolejne słowa dookreślają typ takiego obiektu, ale mogę to zmienić